Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -443,7 +446,7 @@ void qMRMLSubjectHierarchyTreeView::setCurrentItems(vtkIdList* items)
     return;
     }
 
-  this->selectionModel()->clearSelection();
+  this->clearSelection();
```

Comment:
Can you please add a comment why calling this is necessary?

Reply:
I do not know what motivates the clear before the change of items, this was here prior to my changes, I just replaced the `model::clearSelection()` by the wrapped version of that function in that class.

Reply:
[See above] Thanks!